kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/model/RegisteredProperty.kt

```diff
@@ -0,0 +1,13 @@
+package godot.entrygenerator.model
+
+data class RegisteredProperty(
+    val fqName: String,
+    val type: Type,
+    val isMutable: Boolean = true,
+    val isLateinit: Boolean = false,
+    val isOverridee: Boolean = false,
```

Comment:
typo or my broken English ?

Reply:
Actually probably both of us have broken english xD
I wondered that as well. But ksp names it that way and IIRC in PSI there is also a property or function that names it like that. An alternative name in this case would probably be `isOverriding` or something like that.
I'm open for suggestions :-)

Reply:
Maybe a list of modifiers would actually make more sense. So we can replace `isMutable`, `isLateinit` and `isOverridee` with a property which is a list of modifiers.

Reply:
I guess it's similar to cases where we write Clazz

Reply:
WDYM?

Reply:
Well, we don't write  "Class" directly to name stuff because it's already taken as a keyword, that's why we write "Clazz" instead. 
So I was wondering if they were using "Overridee" for the same reason.

Reply:
Ah yeah probably. An additional reason might be because ksp provides a method to get the nearest overridden member. This function is called `findOverridee`. It's probably to match that as well.